docs/descriptor/includes/image/osbs.rst

```diff
@@ -50,6 +49,31 @@ in the :ref:`generation phase<handbook/building/build-process:Generating require
     osbs:
         extra_dir: custom-files
 
+OSBS extra copy command
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Key
+    ``extra_dir_target``
+Required
+    No
+
+If this key is set then the contents of ``extra_dir`` will be copied during the image build to the location specified by this key. If this is *not* set, no ``COPY`` commands will be added. For example, with the value of ``/`` then
```

Comment:
Why not set the default value of the `extra_dir_target` to what is specified in `extra_dir_target` and copy it always to a directory with the same name and optionally changing the target name?